includes/Level.hpp

```diff
@@ -78,6 +83,19 @@ class Level
        we only use dynamic allocation to make the program run faster, so this
        is always better to limit the freedom of variables */
     std::vector<std::unique_ptr<Cell>> cells;
+
+    /* the current player cell index; from 0 to 3199; this variable is not
+       directly initialized here because we set it with the index of the last
+       found departure cell during the level construction */
+    unsigned short playerIndex;
+
+    /* */
+    sf::Int32 lastPlayerCellAnimation {0};
+
+    /* */
+    sf::Uint8 playerCellTransparency {0};
+
+    sf::Uint8 transparencyDelta {17};
```

Comment:
all these items are related to the game only (and not to all the use of the level), should be inside the game controller